vm/src/builtins/super.rs

```diff
@@ -110,10 +110,26 @@ impl PySuper {
     }
 
     #[pyproperty(magic)]
-    fn thisclass(&self) -> PyTypeRef {
+    fn __thisclass__(&self) -> PyTypeRef {
         self.typ.clone()
     }
 
+    #[pyproperty(magic)]
+    fn __self_class__(&self) -> Option<PyTypeRef> {
```

Comment:
As youknowone said, `magic` in `#[pyproperty]` attach two underscores at the left-right sides automatically. This code leads to the below result.
```python
>>>>> super(list).__self_class__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'super' object has no attribute '__self_class__'. Did you mean: '____self_class____'?
```

You can use `#[pyproperty(name = "__self__")]` instead of `magic`. 
The reserved keyword `self` could be replaced with `zelf`.